### ISSUE TITLE
[a11y] Audit Input Date

### ISSUE THREAD

**Issue by samuel-gomez-axa** [AxaFrance/react-toolkit#1156]
# Vérification de l'accessibilité du composant : composant

Afin d'évaluer le niveau d'accessibilité d'un composant, il faut effectuer les actions ci-dessous.

## Analyse statique d'accessibilité

Remplir les champs ci-dessous, en se basant sur l'onglet accessibilité de chaque composant dans le storybook.

**VIOLATIONS** : 

*INCOMPLETE* :

PASSED :

En plus mettre une capture de l'onglet en commentaire de cette issue.

## Analyse manuelle

- [ ] Est ce que le composant est navigable au clavier (tab) ?
- [ ] Est ce que le composant est navigable avec NVDA ?
- [ ] Est ce que le composant est navigable avec un autre lecteur d'écran ?

## Vérification sémantique

- [ ] Vérifier l'implémentation sémantique des éléments html ?
- [ ] Vérifier la validité HTML W3C (https://validator.w3.org/#validate_by_input) ?
- [ ] Vérifier la validité CSS (https://jigsaw.w3.org/css-validator/#validate_by_input) ?

## Problèmes détectés

Si vous avez détectés un défaut d'accessibilité, veuillez créer une issue décrivant le défaut et la rattacher à cette issue.
